benchmarks/gravity_prem/prem.cc

```diff
@@ -0,0 +1,176 @@
+/*
+  Copyright (C) 2011 - 2020 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+#include <aspect/simulator.h>
+#include <aspect/material_model/simple.h>
+#include <aspect/boundary_velocity/interface.h>
+#include <aspect/simulator_access.h>
+#include <aspect/global.h>
```

Comment:
I dont think you need any of these lines. The only one you need is:
```suggestion
 #include <aspect/material_model/interface.h>
```